observability/index.adoc

```diff
@@ -23,9 +23,13 @@ ifndef::openshift-dedicated,openshift-rosa[]
 * {PM-shortname-c}
 endif::openshift-dedicated,openshift-rosa[]
 
-
 {ObservabilityLongName} connects open-source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
 
+[NOTE]
+====
+With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-name}. The Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page outlines their release compatibility.
```

Comment:
what's the value of `product-name`?
from https://80691--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/index.html
```
With the exception of monitoring, Red Hat OpenShift Observability components have distinct release cycles separate from the core {product-name}. The Red Hat [OpenShift Operator Life Cycles](https://access.redhat.com/support/policy/updates/openshift_operators) page outlines their release compatibility.
```
`product-name` is not replaced with a actual name

Reply:
That is a mistake of mine :cry: , thank you for noticing! Should be product-title. I quickly fixed it.